contracts/protocore/Protocore.sol

```diff
@@ -64,14 +64,58 @@ contract Protocore is MosaicVersion, ValidatorSet, CoconsensusModule {
     }
 
 
+    /* Constants */
+
+    /** EIP-712 domain separator name for ProtoCore. */
+    string public constant DOMAIN_SEPARATOR_NAME = "Mosaic-Core";
+
+    /** EIP-712 domain separator typehash for ProtoCore. */
+    bytes32 public constant DOMAIN_SEPARATOR_TYPEHASH = keccak256(
+        "EIP712Domain(string name,string version,bytes32 metachainId,address core)"
+    );
+
+    /** EIP-712 type hash for a Vote Message */
+    bytes32 public constant VOTE_MESSAGE_TYPEHASH = keccak256(
+        "VoteMessage(bytes32 transitionHash,bytes32 sourceBlockHash,bytes32 targetBlockHash,uint256 sourceBlockHeight,uint256 targetBlockHeight)"
```

Comment:
`sourceBlockNumber` and `targetBlockNumber`

@jayeshbairagi can you create a ticket for changing this in `Core.sol` as well? please leave this out of the sprint, noone should implement this during the sprint

Reply:
at least not until we ve completed all the originally pointed tickets; @deepesh-kn please also help enforce this, so that we correctly measure velocity

Reply:
Changed to `sourceBlockNumber` and `targetBlockNumber`.